docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+metadata:
+  name: hello-display
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: hello-display
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: event-display
+          # Source code: https://github.com/knative/eventing-contrib/blob/release-0.6/cmd/event_display/main.go
+          image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:37ace92b63fc516ad4c8331b6b3b2d84e4ab2d8ba898e387c0b6f68f0e3081c4
+
+---
+
+# Service pointing at the previous Deployment. This will be the target for event
+# consumption.
+kind: Service
+apiVersion: v1
+metadata:
+  name: hello-display
+spec:
+  selector:
+    app: hello-display
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+END
+```
+
+2. To create the `goodbye-display` subcomponent, enter the following command:
+
+```sh
+kubectl -n event-example apply -f - << END
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: goodbye-display
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: goodbye-display
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: event-display
+          # Source code: https://github.com/knative/eventing-contrib/blob/release-0.6/cmd/event_display/main.go
+          image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:37ace92b63fc516ad4c8331b6b3b2d84e4ab2d8ba898e387c0b6f68f0e3081c4
+
+---
+
+# Service pointing at the previous Deployment. This will be the target for event
+# consumption.
+kind: Service
+apiVersion: v1
+metadata:
+  name: goodbye-display
+spec:
+  selector:
+    app: goodbye-display
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+END
+```
+
+3. Just like the `Broker`, verify that the event consumers are working with the following command:
+
+```sh
+kubectl -n event-example get deployments hello-display goodbye-display
+```
+
+This commmand shows all of the deployments that have been created in `hello-display` and `goodbye-display`. You should have one replica in each event consumer:
+
+```sh
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+hello-display    1         1         1            1           26s
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+goodbye-display    1         1         1            1           16s
+```
+
+The number of replicas in the **DESIRED** column should match the number of replicas in the **AVAILABLE** column. This may take a few minutes. If after two minutes the numbers still do not match the example, then see the [Debugging Guide](./debugging/README.md).
+
+### Creating `Triggers`
+
+A `Trigger` expresses a workload's interest in events it wants to receive. A `Trigger` is split into two parts: the `Filter`, which tracks interested events, and the `Subscriber`, which determines where the event should be sent. Triggers can search for various types of events:
```

Comment:
I would use "consumer" or "subscriber" instead of "workload" here, since you're using those words elsewhere (but maybe I missed a prior use of workload).

I'd also say the Filter "specifies interesting events". Filter and Subscriber probably should not be capitalized.

Use "match" instead of "search". Match is used elsewhere.